examples/send_and_subscribe.rs

```diff
@@ -0,0 +1,153 @@
+extern crate env_logger;
```

Comment:
This example is no longer its own subcrate, which I'm ok with but it makes it difficult to tell where the diffs are.